docs/user/implement_algo.rst

```diff
@@ -212,10 +212,18 @@ generators used for the experiment.
     debug_my_algorithm()
 
 
+Trainning the Policy with Samples
+=================================
+
 Of course, we'll need to actually use the resulting samples to train our
-policy. For the purposes of this tutorial, we'll implement an extremely simple
-form of REINFORCE [1]_ (a.k.a. Vanilla Policy Gradient) using PyTorch.
+policy with PyTorch, TensorFlow or NumPy. For the purposes of this tutorial,
+we'll implement an extremely simple form of REINFORCE [1]_ (a.k.a. Vanilla
+Policy Gradient) using PyTorch and TensorFlow. If you want to use NumPy, you
+can refer `cem.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/np/algos/cem.py>`_,
+which implements a simple Cross Entropy Method algorithm.
 
+PyTorch
```

Comment:
I think in both cases you should include console output and even plots for what the user should expect to see. Otherwise, they won't have a way of knowing if their example is working.